source/includes/_tipser-sdk.md

```diff
@@ -93,35 +95,27 @@ Sometimes Tipser dialogue styling and functionality needs to be customized. It i
         });
 ```
 
-Hiding icons on menu bar
+###Hiding icons on menu bar
 
 The following parameters under modalUi can be used to selectively hide tipser icons on the dialog menu bar: 
 
-* hideSearchIcon 
-* hideFavouritesIcon
-* hideCartIcon
-* hideMoreIcon
+`hideSearchIcon` <br> 
```

Comment:
`br`s here shouldn't be needed here. Maybe you wanted to leave an empty line between these elements?

Reply:
Yes, but the indentation then is too big, <br> lets us have a new line without one line break.